XenAdmin/Dialogs/VmSnapshotDialog.cs

```diff
@@ -151,7 +151,7 @@ private void pictureBoxQuiesceInfo_Click(object sender, EventArgs e)
                 tt = Messages.FIELD_DISABLED;
             else if (_VM.power_state != vm_power_state.Running)
                 tt = Messages.INFO_QUIESCE_MODE_POWER_STATE.Replace("\\n", "\n");
-            else if (_VM.virtualisation_status != VM.VirtualisationStatus.OPTIMIZED)
+            else if (!_VM.GetVirtualisationStatus.HasFlag(VM.VirtualisationStatus.OPTIMIZED))

```

Comment:
I'm not sure why this changed from virtualisation_status to GetVirtualisationStatus.